app/actors/curation_concerns/actor_stack.rb

```diff
@@ -0,0 +1,27 @@
+module CurationConcerns
+  class ActorStack
+    attr_reader :curation_concern, :user, :first_actor_class, :more_actors
+    def initialize(curation_concern, user, more_actors)
+      @curation_concern = curation_concern
+      @user = user
+      @more_actors = more_actors
+      @first_actor_class = @more_actors.shift || RootActor
```

Comment:
@mjgiarlo I considered that, but the RootActor returns true for all methods, so calling it the NullActor seems confusing.